src/components/MenuItem.js

```diff
@@ -106,6 +106,23 @@ const MenuItem = (props) => {
                                 />
                             </View>
                         )}
+                        {(props.icon && props.iconType === CONST.ICON_TYPE_WORKSPACE) && (
+                            <View
+                                style={[
+                                    styles.popoverMenuIcon,
+                                    ...props.iconStyles,
+                                ]}
+                            >
+                                <Avatar
+                                    imageStyles={[styles.alignSelfCenter]}
+                                    size={CONST.AVATAR_SIZE.DEFAULT}
+                                    source={props.icon}
+                                    fallbackIcon={props.fallbackIcon}
+                                    name={props.title}
+                                    type={CONST.ICON_TYPE_WORKSPACE}
+                                />
+                            </View>
+                        )}
```

Comment:
NAB: Since the `<View>` wrapping is always the same, maybe the code can be drier with something like
```jsx
{props.icon && (
    <View
        style={[
            styles.popoverMenuIcon,
            ...props.iconStyles,
        ]}
    >
        {props.iconType === CONST.ICON_TYPE_ICON && (
            <Icon
                src={props.icon}
                width={props.iconWidth}
                height={props.iconHeight}
                fill={props.iconFill || StyleUtils.getIconFillColor(
                    getButtonState(props.focused || hovered, pressed, props.success, props.disabled, props.interactive),
                    true,
                )}
            />
        )}
        {props.iconType === CONST.ICON_TYPE_WORKSPACE && (
            <Avatar
                imageStyles={[styles.alignSelfCenter]}
                size={CONST.AVATAR_SIZE.DEFAULT}
                source={props.icon}
                fallbackIcon={props.fallbackIcon}
                name={props.title}
                type={CONST.ICON_TYPE_WORKSPACE}
            />
        )}
        {props.iconType === CONST.ICON_TYPE_AVATAR && (
            <Avatar
                imageStyles={[styles.avatarNormal, styles.alignSelfCenter]}
                source={props.icon}
                fallbackIcon={props.fallbackIcon}
            />
        )}
    </View>
)}
```